lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -158,32 +156,39 @@ def get_running_instance_ip
       EcsManager.get_running_instance_ips(config.ecs.cluster, family).fetch(@deploy_config.instance)
     end
 
-    # creates a new task revision using current directory's env vars and provided tag
+    # Creates a new task revision using current directory's env vars, provided tag, and configured options.
+    # Currently can only handle a single container definition.
     def update_task_revision
-      revision = create_new_task_revision
-
-      revision[:container_definitions].select { |c| c[:name] == family }.first.tap do |container_def|
-        container_def[:environment] = @deploy_config.env_vars
-        container_def[:image] = image_tag
-        container_def[:command] = @deploy_config.command
+      revision = EcsManager.get_latest_task_definition(family)
+      if revision[:container_definitions].size != 1
```

Comment:
why can't you look for the container definition that matches the family anymore?